.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.17.0-otp-27
+elixir 1.18.0-otp-27
 erlang 27.0
```

Comment:
Btw. if you're bumping the Erlang version in the docker image to 27.2 for all the tooling, we should do the same in tool-versions